app/controllers/pages_controller.rb

```diff
@@ -77,4 +96,8 @@ def handle_submit_action
   def answer_types
     @answer_types = Page::ANSWER_TYPES
   end
+
+  def mark_complete_options
+    [OpenStruct.new(value: "true"), OpenStruct.new(value: "false")]
+  end
```

Comment:
Does this need to be defined in both `pages_controller` and `page_list_controller`?

Reply:
Good spot, I looked into this and it turned out we didn't even need this or the instance variables which were calling this. These values are actually set inside a view component which expects specific values. a196337b4dec47ecc4e2251e03b9d91fdb76e566 removes the instance var and this method

Reply:
Ah, you mean https://github.com/alphagov/forms-admin/blob/main/app/components/mark_complete_component/?

Reply:
> These values are actually set inside a view component which expects specific values.

Might be worth noting this in the commit message